ci/validate_rsa_keys.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+while read pub_key; do
+  rsa_bit_length=$(echo ${pub_key} | ssh-keygen -lf - | { read -a i; echo ${i[0]}; })
+  if [ ${rsa_bit_length} -lt 2048 ]; then
```

Comment:
Please increase to 4096 bits as recommended by https://infosec.mozilla.org/guidelines/openssh#key-generation. Thanks!

Reply:
see also https://infosec.mozilla.org/guidelines/key_management#acceptable---generally-valid-for-up-2-years (just a few more options)